channels/offline-channel-js/Tests/Unit/src/channel.tests.ts

```diff
@@ -79,6 +81,8 @@ export class ChannelTests extends AITestClass {
                     }
                 });
 
+                this.clock.tick(1);
```

Comment:
Hmm, this triggered an interesting thought (not sure if it true)...

When the SDK is initialized from the snippet it "initializes" and then sends out a "trackPageView" (synchronously) which if we are online is not a problem, but if the offlineListener says we are offline this will try and add the event to the offline batch which its won't be able to do because the "timer" has not yet been triggered (because everything from initialization -> trackPageview is synchronous) so this could actually cause the lose of this event....

Let me go back and look at the offline channel, we should be able to rework fairly simply to get rid of the timer (I think) and only perform the "lazy" initialization (if) required.....

Reply:
As discussed there is already the check for _offlineSupport in processtelemetry that will handle this case